data/weapons.txt

```diff
@@ -838,7 +838,7 @@ outfit "Torpedo Launcher"
 		"hull damage" 680
 		"hit force" 300
 		"missile strength" 30
-	description "Torpedoes are the most powerful homing weapon, but they move slowly enough that fast ships can simply outrun them, making them most useful against large targets or against small ships at times when they are too distracted to dodge. Torpedo launchers are also much larger than other missile systems, so they are most often found in capital ships."
+	description "Torpedo Launchers are the largest and most powerful homing weapons, often found on capital ships for fleet engagements. Fast ships can often simply outrun or dodge the Torpedoes they fire, making them generally effective only against large slow targets. However, very skillful pilots have been known to score hits on smaller ships, especially at close range."
```

Comment:
I don't think that any of these weapon description changes are really necessary. They're largely just rephrasing what was already said.